contracts/modules/v2/StakingPool.sol

```diff
@@ -450,6 +544,19 @@ contract StakingPool is IStakingPool, ERC721 {
     activeStake = _activeStake > amount ? _activeStake - amount : 0;
   }
 
+  /* nft */
+
+  function _beforeTokenTransfer(
+    address from,
+    address /*to*/,
+    uint256 tokenId
+  ) internal view override {
+    require(
+      tokenId != 0 || nxm.isLockedForMV(from) < block.timestamp,
```

Comment:
Why would token 0, namely the manager be exempted from governance locking? Isn't the manager's address the one that's locked?

My concern here is that someone who has access to multiple member addresses can transfer the token 0 and vote multiple times with the balance of the entire pool.

Reply:
You understood it backwards. The code ensures that the id of the token being transfered is not zero, or if it is - it's not locked for voting in governanace.

Reply:
I think I understood it exactly as you've said. If the token that is being transfered is zero, it's "not" locked for governance voting, hence it can be transfered to another account and used to vote with the whole pool worth of NXM again.

Reply:
`nxm.isLockedForMV(from)` returns the timestamp until which the user is locked.

The require uses this condition:
`tokenId != 0 || nxm.isLockedForMV(from) < block.timestamp`

1) If the token id is NOT ZERO, the condition passes immediatelly. If it IS ZERO, the first part will be false so it will evaluate the second part.
2a) If the user voted in governance recently then `nxm.isLockedForMV(from)` will be in the future, so `nxm.isLockedForMV(from) < block.timestamp` will be false, which would make the require statement to revert, which is the intended behaviour.
2b) If the user did not vote in governance recently then `nxm.isLockedForMV(from)` will be less than the current block timestamp so the expression will be true, so the require will pass without reverting.